geaflow/geaflow-plugins/geaflow-store/README.md

```diff
@@ -0,0 +1,130 @@
+# GeaFlow Store API
+
+GeaFlow Store API provides a unified storage interface for managing state data in graph computing.
+This document introduces how to use and implement these storage interfaces.
+
+## Core Interfaces
+
+GeaFlow Store API includes the following core interfaces:
+
+### IStoreBuilder
+
+Interface for building storage instances. Main methods:
+
+- `getStoreDesc()`: Get storage description
+- `build()`: Build storage instance
+
+### IBaseStore
+
+The base interface for all storage implementations, defining basic storage operations:
+
+- `init()`: Initialize storage
+- `flush()`: Flush data to storage
+- `close()`: Close storage connection
+
+### IStatefulStore
+
+State management storage interface, inheriting from IBaseStore, providing stateful management
+operations:
+
+- `archive()`: Archive current state
+- `recovery()`: Recover state from specified position
+- `recoveryLatest()`: Recover state from specified position
```

Comment:
The descriptions for the recoveryLatest and recovery interfaces should be distinguished from each other.

Reply:
@Loognqiang issue has been resolved.